build.gradle

```diff
@@ -128,6 +136,9 @@ license {
 }
 
 test {
+
+    useJUnitPlatform()
```

Comment:
I'm not a gradle user, so maybe there's a better way, but without this config I couldn't get the JUnit-5 tests recognised.